### PR TITLE
WEBDEV-4557 - Fix guest user dropdown position when there is no search bar in topnav

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-4557

### CODE REVIEW

File: packages/ia-topnav/src/ia-topnav.js [hunk 1]
```diff
@@ -172,6 +172,7 @@ export default class IATopNav extends LitElement {
         .baseHost=${this.baseHost}
         .config=${this.config}
         .open=${this.signedOutOpened}
+        ?hideSearch=${this.hideSearch}
```
Comment (jbuckner) [inline packages/ia-topnav/src/ia-topnav.js:175]: Q: Is this part of the fix or is this something else?

File: packages/ia-topnav/src/styles/signed-out-dropdown.js [hunk 2]
```diff
@@ -5,27 +5,27 @@ export default css`
...
     .open {
-      right: 39.7rem;
+      right: 40rem;
     }
 
     .search-hidden.initial,
     .search-hidden.closed,
     .search-hidden.open {
-      right: 23.5rem;
+      right: 24.5rem;
```
Comment (jbuckner) [inline packages/ia-topnav/src/styles/signed-out-dropdown.js:28]: Q: It seems like these adjustments are only a few pixels each, but the error looks like it's offset by 100-200px. How are they shifting that menu over so far?

**Review by jbuckner — APPROVED** [review]
Two questions, but I tested the demo and it looks good. 👍

**Comment by jbuckner** [comment]
Oh I see what happened! The search wasn't showing when it was supposed to and we were offsetting assuming it was visible.

**Comment by nsharma123** [comment]
> Oh I see what happened! The search wasn't showing when it was supposed to and we were offsetting assuming it was visible. Looks good

Yeah, the original fix is just passing `?hideSearch=${this.hideSearch}` prop in `<signed-out-dropdown>` component. The small change is  rem units just to show dropdown arrow more correctly.

File: packages/ia-topnav/src/ia-topnav.js (hunk 1 above)
Comment (nsharma123) [reply packages/ia-topnav/src/ia-topnav.js:175]: Yes

**Pull request merged**